src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -3043,8 +3084,10 @@ def oil_water_surface_tension_model(p, t):
 
             names += [
                 "water density",
-                "water density derivative pressure",
-                "water density derivative temperature",
+                "water density derivative pressure, const temperature",
+                "water density derivative temperature, const pressure",
+                "water density derivative pressure, constant enthalpy",
+                "water density derivative enthalpy, constant pressure",
```

Comment:
If we got alfacase files saved with the old names they will be loaded with the old names.
Today we have no support for backward compatibility in alfacase files.
Will this be a problem?

Reply:
As we discussed via zulip, I created this issue to investigate this point - ASIM-5482